lib/binary_to_decimal.rb

```diff
@@ -5,5 +5,18 @@
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
+  binary_base_2 = [ 128, 64, 32, 16, 8, 4, 2, 1 ]
```

Comment:
Minor issue:  This definitely works, but you've had to hardcode all the powers of 2 involved.  Any way to make this more flexible?